### PR TITLE
CI: Add C++ and Python formatting as build checks

### PR DESCRIPTION
**Pull request by sentree**
This adds a stage to the end of the build process that checks whether or not the new code is formatted according to our style guides. The build will fail if the files are not formatted correctly and a list of the ones that need to be changed for each language will be printed. See the [mil_common wiki](https://github.com/uf-mil/mil_common/wiki) for more information on how to automatically format code.

Have a look at [this Jenkins build](https://ci.mil.ufl.edu/jenkins/blue/organizations/jenkins/uf-mil%2Fmil_common/detail/PR-55/3/pipeline) for this for information on what Python files need to be fixed.

Have a look at [this Jenkins build](https://ci.mil.ufl.edu/jenkins/blue/organizations/jenkins/uf-mil%2Fmil_common/detail/PR-55/2/pipeline) for this for information on what C++ files need to be fixed. @DSsoto, this is all yours if you want it.

Ignore issues in ros_alarms and txros for now. These will be fixed once the formatting pull requests on those repositories are merged and the submodules are updated in this one.

As each formatting pull request is merged to master, the build of this pull request should be run again by the user. Once all of the formatting checks pass, we can merge this. If you disagree with one of the formatting rules, feel free to debate it on this pull request and I will consider removing it.

### CODE REVIEW

**Comment by RustyBamboo** [comment]
In order to avoid segregation, I believe a formatting check should be also done on C, Cmake, Yaml, json, XML, and, in particular, shell and bash...     ;)

**Comment by DSsoto** [comment]
@RustyBamboo looking forward to seeing your proposed methods for checking all of these auxillary languages ;)

**Comment by sentree** [comment]
It is totally possible to check these languages, so long as @RustyBamboo finds a program to check them, finds a style guide everyone agrees on, adds the checker to the Docker image and Jenkins file, tests the changes, and formats all of the old code. Personally, my Bash style is pretty standardized, so no need to worry about formatting that. Enjoy!

**Comment by sentree** [comment]
These changes are complete. In order for formatting checks to pass txros/txros#23 and #17 need to be merged. @DSsoto should also make sure that everything in `drivers/mil_passive_sonar` is properly formatted in #17 before that is merged.

**Comment by sentree** [comment]
This is good to merge now.

File: perception/mil_vision/include/camera_lidar_transformer.hpp [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #pragma once
-#include <geometry_msgs/TransformStamped.h>
```
Comment (DSsoto) [inline perception/mil_vision/include/camera_lidar_transformer.hpp:2]: Good stuff @whispercoros!

**Review by DSsoto — APPROVED** [review]

**Pull request merged**